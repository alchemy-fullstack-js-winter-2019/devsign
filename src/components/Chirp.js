import React from 'react';
import styled from 'styled-components';

const Section = styled.section` 
  margin-top: 0;
  display: flex;
  h4 {
    font-weight: 300;
    margin: 0;
  }
  .info {
    display: flex;
    flex-direction: column;
  }
  img {
    margin-right: 10px;
  }
  .handle {
    margin: 0 0 10px 0;
  }
`;
export default function Chirp() {
  return (
    <Section>
      <img src="../../assets/prof-pic.png" alt="prof-pic" height="40px"/>
      <div className="info">
        <h4>UserName</h4>
        <p className="handle">@userhandle</p>
        <p>My name is somewhat extravagant, and I want to cut back. It's the good costumes, all year long. #mybite #randomtweet</p>
      </div>
    </Section>
  );
}
