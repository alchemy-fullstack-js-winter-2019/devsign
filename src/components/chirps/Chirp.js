import React from 'react';
import styled from 'styled-components';

const Section = styled.section` 
  width: 90vw;
  margin-top: 0;
  display: flex;
  h4 {
    font-weight: 300;
    margin: 0;
  }
  .info {
    display: flex;
    flex-direction: column;
    width: 70vw;
  }
  .body {
    width: 55vw;
  }
  img {
    margin-right: 10px;
  }
  .handle {
    margin: 0 0 10px 0;
  }
`;
export default function Chirp() {
  return (
    <Section>
      <img src="../../assets/prof-pic.png" alt="prof-pic" height="40px"/>
      <div className="info">
        <h4>UserName</h4>
        <p className="handle">@userhandle</p>
        <p className="body">My name is somewhat extravagant, and I want to cut back. Its the good costumes, all year long. #mybite #randomtweet</p>
      </div>
    </Section>
  );
}
