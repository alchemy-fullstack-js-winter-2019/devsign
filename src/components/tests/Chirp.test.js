import React from 'react';
import renderer from 'react-test-renderer';
import Chirp from '../chirps/Chirp';

describe('Chirp', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <Chirp />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
