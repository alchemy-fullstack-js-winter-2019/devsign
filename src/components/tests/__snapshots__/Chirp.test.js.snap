// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chirp matches a snapshot 1`] = `
<section
  className="sc-bdVaJa gyTtzj"
>
  <img
    alt="prof-pic"
    height="40px"
    src="../../assets/prof-pic.png"
  />
  <div
    className="info"
  >
    <h4
      className="name"
    >
      UserName
    </h4>
    <p
      className="handle"
    >
      @userhandle
    </p>
    <p
      className="body"
    >
      My name is somewhat extravagant, and I want to cut back. Its the good costumes, all year long. #mybite #randomtweet
    </p>
  </div>
</section>
`;
